runtime/integration-tests/src/cases/lp/transfers.rs

```diff
@@ -271,7 +272,9 @@ fn transfer_tranche_tokens_domain_to_local_to_domain<T: Runtime>() {
 				Token::FixedBytes(pool_a_tranche_1_id::<T>().into()),
 				Token::Uint(DOMAIN_EVM.into()),
 				Token::Uint(EVM_DOMAIN_CHAIN_ID.into()),
-				Token::Address(Keyring::TrancheInvestor(2).into()),
+				Token::FixedBytes(
+					AccountId::from(as_h160_32bytes(Keyring::TrancheInvestor(2))).to_raw_vec(),
```

Comment:
Damn, you were faster! Just came to the same conclusion after some debugging. Well done!

Reply:
Happy to reach the same solution, double confident about this!